packages/slice-machine/pages/onboarding.tsx

```diff
@@ -140,29 +147,126 @@ const StepIndicator = ({
   );
 };
 
+function idFromStep(
+  step: number
+):
+  | TrackingEventId.ONBOARDING_CONTINUE_SCREEN_INTRO
+  | TrackingEventId.ONBOARDING_FIRST
+  | TrackingEventId.ONBOARDING_SECOND
+  | TrackingEventId.ONBOARDING_THIRD {
+  switch (step) {
+    case 0:
+      return TrackingEventId.ONBOARDING_CONTINUE_SCREEN_INTRO;
+    case 1:
+      return TrackingEventId.ONBOARDING_FIRST;
+    case 2:
+      return TrackingEventId.ONBOARDING_SECOND;
+    default:
+      return TrackingEventId.ONBOARDING_THIRD;
+  }
+}
+
+function postTracking(
+  data:
```

Comment:
we should avoid using generic name as data, stuff, things, etc when we can, I was thinking about "onboardingEvent" to be more specific, what do you think about it ?